carma-messenger-ui/website/widgets/emergencyResponse/widget.js

```diff
@@ -81,6 +118,23 @@ var subscribe_bsm = () => {
         }
     });
 }
+var createFeature = () => {
```

Comment:
Let's add comments to describe the `createFeature` and `createMarker` functions as well.

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/pull/177/commits/89771b758699a93896e5555ceb112d4a9e320309